x-pack/legacy/plugins/xpack_main/server/lib/__tests__/xpack_info.js

```diff
@@ -383,18 +158,20 @@ describe('XPackInfo', () => {
 
   describe('feature', () => {
     let xPackInfo;
+    let license$;
```

Comment:
I removed all fetching tests. All remaining tests in the file are supposed to prove the xpack plugin contract.